java/src/com/facebook/watchman/WatchmanConnection.java

```diff
@@ -45,7 +45,7 @@
 
   private final ListeningExecutorService outgoingMessageExecutor;
   private final ExecutorService incomingMessageExecutor;
-  private final Supplier<Map<String, Object>> inputMessageSupplier;
+  private final Callable<Map<String, Object>> inputMessageCallable;
```

Comment:
It looks like this rename hides the meaning of this variable. This is something that provides input messages. The new name looks like it's something that is called on an input message or something similar, but not the original purpose.